gallery/src/pages/lovelace/entity-filter-card.ts

```diff
@@ -107,6 +132,43 @@ const CONFIGS = [
     title: Custom Title
     `,
   },
+  {
+    heading: "Unfiltered number entities",
+    config: `
+- type: entities
+  entities:
+    - sensor.gas_station_auchan_saint_priest_e10
+    - sensor.gas_station_carrefour_venissieux_e10
+    - sensor.gas_station_relais_lyon_mermoz_e10
+    - sensor.gas_station_lowest_price
+    `,
+  },
+  {
+    heading: "Filtered entities with operator & number value",
+    config: `
+- type: entity-filter
+  entities:
+    - sensor.gas_station_auchan_saint_priest_e10
+    - sensor.gas_station_carrefour_venissieux_e10
+    - sensor.gas_station_relais_lyon_mermoz_e10
+  state_filter:
+    - operator: <=
+      value: 1.73
+    `,
+  },
+  {
+    heading: "Filtered entities with operator & entity_id value",
+    config: `
+- type: entity-filter
+  entities:
+    - sensor.gas_station_auchan_saint_priest_e10
+    - sensor.gas_station_carrefour_venissieux_e10
+    - sensor.gas_station_relais_lyon_mermoz_e10
+  state_filter:
+    - operator: ==
+      value: sensor.gas_station_lowest_price
+    `,
+  },
```

Comment:
```suggestion
  },
  {
    heading: "Filtered entities with operator & template value",
    config: `
- type: entity-filter
  entities:
    - sensor.gas_station_auchan_saint_priest_e10
    - sensor.gas_station_carrefour_venissieux_e10
    - sensor.gas_station_relais_lyon_mermoz_e10
  state_filter:
    - operator: <=
      value: {{ float(states.sensor.gas_station_lowest_price.state) + 0.05 }}
    `,
  },
```
I would love to make `value` template compatible too but I couldn't make it.

Having `filter.value` = `[object Object]: null`
![image](https://github.com/home-assistant/frontend/assets/14821482/3d4a1e1a-9645-4f7e-b2a2-5900fb7e9457)

Reply:
I believe it's a conscious decision to not really support templates in any frontend cards (aside from maybe markdown).

I think it might be a performance thing, but I could be wrong. 

But I would guess a template is not going to be accepted here.

Reply:
And the sensor itself could be a template one 🤷🏼‍♂️